modin/pandas/test/test_series.py

```diff
@@ -2201,7 +2233,14 @@ def test_mean(data, skipna):
 )
 def test_median(data, skipna):
     modin_series, pandas_series = create_test_series(data)
-    df_equals(modin_series.median(skipna=skipna), pandas_series.median(skipna=skipna))
+    try:
+        pandas_result = pandas_series.median(skipna=skipna)
+    except Exception as e:
+        with pytest.raises(type(e)):
+            modin_series.median(skipna=skipna)
+    else:
+        modin_result = modin_series.median(skipna=skipna)
+        df_equals(modin_result, pandas_result)
```

Comment:
```suggestion
    eval_general(*create_test_series(data), lambda df: df.median(skipna=skipna))
```

The same comments as for `test_all` func

Reply:
Fixed.